internal/controller/templatechain_controller.go

```diff
@@ -113,6 +113,13 @@ func (r *TemplateChainReconciler) ReconcileTemplateChain(ctx context.Context, te
 		return ctrl.Result{}, nil
 	}
 
+	// NOTE (zeropspiel): should we either introduce status with {valid, warnings} or update the chains with some error label?
```

Comment:
I think, yes, when the webhook is disabled, it makes sense to have a `status` with `valid` and `warnings` fields. Initially, we didn't add it since an invalid chain simply couldn't be created due to admission validation.